src/crimson/os/seastore/cached_extent.h

```diff
@@ -1258,6 +1258,14 @@ class ExtentIndex {
     return extent_index.find(offset, paddr_cmp());
   }
 
+  bool exists(CachedExtent& extent) const {
```

Comment:
Should we encapsulate the method with `#ifndef NDEBUG`, since it's cpu consuming and for assertions only?